core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -199,9 +198,15 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
         .registerContributorDashboardViewSuggestionForReview('Translation');
       this.heading = 'Review Translation Contributions';
     }
+    const suggestionIdsList = Object.keys(this.suggestionIdToContribution);
+    const clickedSuggestionIndex = suggestionIdsList.indexOf(
+      this.activeSuggestionId);
+    this.skippedContributionIds.push(...suggestionIdsList.slice(
+      0, clickedSuggestionIndex));
```

Comment:
Can we just reassign `this.skippedContributionIds` instead of appending? Safer to not mutate the object in case `ngOnInit()` gets called multiple times somehow and easier to reason about.

Reply:
Done.